tests/cases/compiler/esModuleInteropImportTSLibHasImport.ts

```diff
@@ -0,0 +1,20 @@
+// @esModuleInterop: true
+// @importHelpers: true
+// @noEmitHelpers: true
+// @target: es2017
+// @module: commonjs
+// @filename: types.d.ts

```

Comment:
Is it necessary for the imported files to actually exist? Would expect the emit to behave the same even if imports are unresolved.

Reply:
I'm a fan of not having a `.error` baseline unless it's important for the test 🤷‍♀️